msteams-platform/concepts/build-and-test/app-studio-overview.md

```diff
@@ -86,8 +86,6 @@ After you have supplied your bot's information, you can now optionally define a
 
 This section allows you to add a connector to your app. If you already have registered an Office 365 connector, choose **Set up** and enter the name and ID of the connector. If you want a new connector click **Register** to be taken to the Connector Developer Dashboard in your browser.
 
-> [!NOTE]
-> App customization enables admins to change the look-and-feel of the apps loaded through bots, messaging extensions, tabs, and connectors. For example, if the Teams admin customizes the name of an app from **Contoso** to **Contoso Agent**, then the app will appear with the new name **Contoso Agent** to users. However, while adding a connector to a chat, in the list the connectors will still show the name of the app as **Contoso**.
```

Comment:
@Mahesh2516 why is this note being removed?

Reply:
@Mahesh2516 It is best practice to not duplicate content. This topic is intended to be an overview of App Studio, while the app customization feature is covered in a separate, dedicated topic.

Reply:
Sure, thanks Heath.